### DISCUSSION TITLE
Travel Distances in Lahar Risk Zones around Mt. Rainier

### DISCUSSION THREAD

**Discussion by Conibaer** [jakobzhao/geog495#72] [Final Project]
**Link:** https://conibaer.github.io/MobilityInLaharRiskAreasAroundMtRainier/index.html
**GitHub:** https://github.com/Conibaer/MobilityInLaharRiskAreasAroundMtRainier
**Description:** This interactive Web application allows the user to investigate  and explore transportation abilities in at risk communities from lahar flows emanating from Mt. Rainier. When the user clicks anywhere on the map and an isochrone polygon is rendered from that selected location. The polygon is based on the travel mode and time parameters selected by the user. The travel distances can then be compared to the at risk area to determine general transportation requirements to reach safety in case of a lahar event. Public schools that fall within the risk zones are also included as they are community centers of special consideration with heightened mobility concerns in the case of a rapidly emerging emergency situation. The schools are a good place to start but the user has the freedom to explore any point on the map they would like. The user can also hover over School locations to activate an information popup

The data is from WA State geospatial portal and the USGS Volcano Hazards Program. I took both data sets into QGIS where I created a true polygon out of the lahar risk area and ran a clip query to keep only the public schools that fell inside the at risk area. The base map was designed by me in the MapBox Style editor in order to maximize topography which plays a crucial role in understanding lahar flow patterns and evacuation challenges.

![Screen Shot 2021-12-11 at 8 12 32 PM](https://user-images.githubusercontent.com/91756808/145700654-1f25fb15-9ccd-408f-ae38-57e652ef5e5f.png)

**Comment by NahomAgidew** [comment]
Hey 👋 You did an amazing job with your map. The idea was creative, and the base map fits very well. I also like how you added personal touches to your README. I would suggest adding custom icons. By the way, I see a legend in your screenshot, but on my end, I don't see it when I open your application.
![image](https://user-images.githubusercontent.com/11695305/145707967-ce341f3a-f741-466e-a9f9-16245dab9085.png)

**Comment by Subarashiihibi** [comment]
Hi, I really like your basemap and layers are just awesome! I am really sure this map would be of help to those who will go hiking in the area.  
I think something that you could improve might be that the range people can walk or drive may be dependent on the landscape of local. And I think some ridges people will take really long time or it is even impossible to climb sheer ridges.  But I still love your map and really cool pictures in your readme.md file!

**Comment by ebar23** [comment]
This has to be one of the coolest maps on here. I think that it is important to know more about Rainer and the damages that can be done as a result of an eruption. This reminds me of the first project you do in GEOG 360 with the lava flows in HI. Honestly, there isn't much that I would change. I might suggest adding a small explanation pop-up of how the map works since I can see how it might be confusing for users who don't read the README file. Other than that though, you did a great job!

**Comment by Stephenwzy** [comment]
This map application is so cool! Since I also double major in ESS, I agree that lahar risk around Mt. Rainier is really an important issue to consider about. I really like this topic. The basemap is also very clear. I agree with what ebar23 said that it is better to include an introductory box in the map, which gives more backgrounds or the main idea of the app.

**Comment by whalen323** [comment]
Wow, this map is so cool! I love the layer of Mount Rainier that you used, and the clear danger level colors also are easy to follow. It's also cool how you incorporated some QGIS capabilities into this map, and made the legend interactive! Honestly the only thing I can find to critique a bit is that for me at least the color shown in the legend as matching areas at risk for lahars doesn't quite match, the map displays as more pink than red. But overall, awesome job!

**Comment by VkSath00** [comment]
This map is so interesting, the layers and basemaps in this application for Mount Rainier are really nice. The color scheme was nice too since displaying where there are potential dangers in the case of a volcanic eruption. I don't think there's much to change, but maybe you could let the user know what the application does with a little intro box.

**Comment by Anemnox** [comment]
Hey! I really like the basemap that you chose for this application. It helps a lot with the terrain and helps define the danger zones from lahar flows. I think one possible thing to make easier to see are the roads on the basemap. It would be helpful to show the paths that are considered from your transportation distance functionality.
